Source/WPF/MyMoney/MainWindow.xaml.cs

```diff
@@ -274,6 +274,23 @@ private void OnSettingsChanged(object sender, System.ComponentModel.PropertyChan
             }
         }
 
+        private void ApplyDefaultCurrency()
+        {
+            // Start with the default USA currency
+            var countryCurrencyCode = "en-USD";
+            decimal convertionRate = 1;
+
+            // Lookup the users App default display currency
+            Currency c = this.myMoney.Currencies.FindCurrency(this.databaseSettings.DisplayCurrency);
+            if (c != null)
+            {
+                countryCurrencyCode = c.CultureCode;
+                convertionRate = c.Ratio;
+            }
+            this.myMoney.Rate = convertionRate;

```

Comment:
I'd prefer to store the default whole Currency object on this.myMoney rather than just the conversionRate because then the reports can pick that up and add this to the bottom:

`Generated in Canadian (CAD) currency on Saturday, February 18, 2023`